src/ert/gui/simulation/view/realization.py

```diff
@@ -144,6 +153,9 @@ def _paint_inner_grid(self, painter: QPainter, rect: QRect, colors) -> None:
         else:
             foreground_image = _image_cache[colors_hash]
 
+        border_pen.setWidth(2)
+        painter.setPen(border_pen)
+        painter.drawRect(rect)
```

Comment:
Why pass the pen? We could avoid doing so?
```
pen = QPen(QColorConstants.Grey)
pen.setWidth(2)
painter.setPen(pen)
painter.drawRect(rect)
```

Also I realised that the default pen is black and width (1).
So we could replace this;

```
        border_pen = QPen()
        border_pen.setColor(QColorConstants.Black)
        border_pen.setWidth(1)
```

Reply:
Are you sure about the default pen? Removing the border_pen results in a white border for me.

Reply:
I removed the pen from the function arguments. I also initialized the pens in-place, maybe it is a bit harder to read?